src/main/java/me/lucien/minesweeper/domain/RoomManager.java

```diff
@@ -0,0 +1,69 @@
+package me.lucien.minesweeper.domain;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class RoomManager {
+
+    @Value("${room.capacity}")
```

Comment:
Set a default value in the program as well. Just in case the app is started with --config=<config file> and the config file contains fewer parameters.